src/main/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/math/Fraction.java

```diff
@@ -22,27 +25,20 @@
 	// should be only called if denom is positive and num & denom are coprime
 	private Fraction(int numerator, int denominator) {
 		assert denominator > 0 : "invalid denominator (must be positive)";
```

Comment:
Are we really trying to use assert here? It won't check normally though.

Reply:
I know, was wondering that too, I didn't write it don't blame me :tiny_potato: I'll fix it though

Reply:
haha thanks